main.py

```diff
@@ -1,3 +1,3 @@
-usernumber = 1
+usernumber1 = 1
```

Comment:
please make this variable name in a snake case format from "usernumber1" to "user_number_1"